pkg/virt-controller/network/multus_annotations.go

```diff
@@ -114,6 +114,12 @@ func newBindingPluginMultusAnnotationData(kvConfig *v1.KubeVirtConfiguration, pl
 	}, nil
 }
 
+func newMultusAnnotationDataWithIPAMClaim(namespace string, interfaces []v1.Interface, network v1.Network, podInterfaceName string, ipamClaimName string) networkv1.NetworkSelectionElement {
+	networkSelectionElement := newMultusAnnotationData(namespace, interfaces, network, podInterfaceName)
+	networkSelectionElement.IPAMClaimReference = ipamClaimName
+	return networkSelectionElement
+}
+
```

Comment:
Why both `newMultusAnnotationDataWithIPAMClaim` and `newMultusAnnotationData` are needed?
Is the any use for `newMultusAnnotationData` besides in `newMultusAnnotationDataWithIPAMClaim`?

Reply:
Nope, @maiqueb preferred that way
imho indeed i would just merge them

Reply:
I think it did make it more explicit to have separate methods for this, but I am forced to say I had understood we would also have to invoke `newMultusAnnotationData` somewhere (e.g. when the network configuration does **not** have the persistent IPs feature enabled). 

If you only have a single call, maybe it does make sense to have a single method.

Reply:
Done